astropy/modeling/core.py

```diff
@@ -695,6 +695,9 @@ class Model(metaclass=_ModelMeta):
     _cov_matrix = None
     _stds = None
 
+    def __init_subclass__(cls, **kwargs):
+        super().__init_subclass__()
+
```

Comment:
Just to clarify, if you pass keyword arguments to a class, do they get passed to `__init_subclass__` too?  This is just a use case I' haven't seen before.  I would have assumed that if that's the case then the default `__init_subclass__` also takes kwargs but maybe not (hence the need for this).

Reply:
yeah, fortunately or unfortunately the default `__init_subclass__` doesn't take a kwarg. This isn't a problem unless the class has a custom meta.